docs/releasenotes/10.2.0.rst

```diff
@@ -20,10 +20,14 @@ ImageFile.raise_oserror
 error codes returned by a codec's ``decode()`` method, which ImageFile already does
 automatically.
 
-TODO
-^^^^
-
-TODO
+IptcImageFile helper functions
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The functions ``IptcImageFile.dump`` and ``IptcImageFile.i``, and the constant
+``IptcImageFile.PAD`` have been deprecated and will be removed in Pillow
+12.0.0 (2025-10-15). These are undocumented helper functions intended
+for internal use, so there is no replacement. They can each be replaced
+by a single line of code using builtin functions in Python.
```

Comment:
Is the idea to a) eventually rename them with underscores and keep using them internally, or b) remove them altogether?

If a) let's add underscore versions and have the public ones issue a warning then call those, and use the underscore versions internally.

If b) let's use the builtins directly now, and only call the deprecated versions from tests.

In both cases, no need for the `_internal: bool` parameter.

Reply:
Both, a) for `i` and b) for `dump` (which is unused otherwise).

> In both cases, no need for the _internal: bool parameter.

Good point, removed.